docs/cloud/manage/organize-your-infrastrucutre-invite-your-team.md

```diff
@@ -0,0 +1,169 @@
+# Organize Your Infrastructure and Invite your Team
+
+Netdata Cloud provides you with features such as [Spaces](#netdata-cloud-spaces) and [War Rooms](#netdata-cloud-war-rooms) that allow you to better organize your infrastructure and ensure your team can also have access to it through invites.
+
+## Netdata Cloud Spaces
+
+Organize your multi-organization infrastructure monitoring on Netdata Cloud by creating Spaces to completely isolate access to your Agent-monitored nodes.
+
+A Space is a high-level container. It's a collaboration space where you can organize team members, access levels and the
+nodes you want to monitor.
+
+Let's talk through some strategies for creating the most intuitive Cloud experience for your team.
+
+### How to organize your Netdata Cloud
+
+You can use any number of Spaces you want, but as you organize your Cloud experience, keep in mind that _you can only
+add any given node to a single Space_. This 1:1 relationship between node and Space may dictate whether you use one
+encompassing Space for your entire team and separate them by War Rooms, or use different Spaces for teams monitoring
+discrete parts of your infrastructure.
+
+If you have been invited to Netdata Cloud by another user by default you will able to see this space. If you are a new
+user the first space is already created.
+
+The other consideration for the number of Spaces you use to organize your Netdata Cloud experience is the size and
+complexity of your organization.
+
+For small team and infrastructures we recommend sticking to a single Space so that you can keep all your nodes and their
+respective metrics in one place. You can then use
+multiple [War Rooms](#netdata-cloud-war-rooms)
+to further organize your infrastructure monitoring.
+
+Enterprises may want to create multiple Spaces for each of their larger teams, particularly if those teams have
+different responsibilities or parts of the overall infrastructure to monitor. For example, you might have one SRE team
+for your user-facing SaaS application and a second team for infrastructure tooling. If they don't need to monitor the
+same nodes, you can create separate Spaces for each team.
+
+### Navigate between spaces
+
+Click on any of the boxes to switch between available Spaces.
+
+Netdata Cloud abbreviates each Space to the first letter of the name, or the first two letters if the name is two words
+or more. Hover over each icon to see the full name in a tooltip.
+
+To add a new Space click on the green **+** button . Enter the name of the Space and click **Save**.
+
+![Switch between Spaces](/img/cloud/main-page-add-space.png)
```

Comment:
```suggestion
![Switch between Spaces](https://github.com/netdata/netdata/assets/70198089/aa0d7a2f-02ec-4c01-a2d9-1f99642f2496)
)
```

That image is not rendering, but I guess we need something like the suggested image